.github/workflows/build.yml

```diff
@@ -15,20 +15,25 @@ jobs:
     - uses: actions/checkout@v3
     - name: dependencies
       run: |
-        sudo apt update
-        sudo apt install libgtest-dev libgflags-dev openssl libssl-dev protobuf-compiler
+        sudo apt update -y
+        sudo apt install -y libgtest-dev libgflags-dev openssl libssl-dev protobuf-compiler protoc-gen-go golang-go
     - name: make
       run: |
+        #! ---------------------------------------------------------------------
         #! Check that core certifier programs still compile
         cd src
+        make -f certifier_tests.mak clean
         make -f certifier.mak
-        rm ./*.o
+        make -f certifier.mak clean
         make -f certifier_tests.mak
         ./certifier_tests.exe
-        #! Check that utilities programs still compile
-        cd ../utilities
-        make -f cert_utility.mak
-        make -f policy_utilities.mak
+        #! ---------------------------------------------------------------------
+        #! Compile and build simple_app. Execute script to exercise simple_app
+        #! This will also check that utilities programs still compile
+        cd ../sample_apps/simple_app
+        ./cleanup.sh
+        ./run_example.sh
```

Comment:
This takes-off in CI but the build fails. See this output:

https://github.com/vmware-research/certifier-framework-for-confidential-computing/actions/runs/4836153662/jobs/8619125922?pr=48

```
+ go build simpleserver.go
go: downloading github.com/golang/protobuf v1.5.2
go: downloading google.golang.org/protobuf v1.28.1
# github.com/jlmucb/crypto/v2/certifier-framework-for-confidential-computing/certifier_service/certlib
certlib/certlib_proofs.go:250:29: undefined: oeverify
certlib/certlib_proofs.go:253:29: undefined: oeverify
```